redbeams/src/main/resources/schema/app/20201013110628_CB-9289_ssl_certificate_added_to_db_stack.sql

```diff
@@ -0,0 +1,30 @@
+-- // CB-9289 ssl certificate added to db stack
+-- Migration SQL that makes the change goes here.
+
+CREATE TABLE sslconfig
+(
+    id BIGINT NOT NULL,
+    PRIMARY KEY (id)
+);
+
+CREATE SEQUENCE sslconfig_id_seq START WITH 1 INCREMENT BY 1 NO MINVALUE NO MAXVALUE CACHE 1;
+
+CREATE UNIQUE INDEX IF NOT EXISTS sslconfig_id_idx ON sslconfig(id);
+
+CREATE TABLE sslconfig_sslcertificates (
+    sslconfig_id bigint NOT NULL REFERENCES sslconfig (id),
+    sslcertificate_value text,
+    type VARCHAR(255) DEFAULT 'CLOUD_PROVIDER_OWNED'
```

Comment:
1. I think this column is part of the `sslconfig` table;  please move it there.
2. Please rename it to `sslcertificatetype`.
3. Why not restrict it to `NOT NULL` (and then `DEFAULT` would not be necessary)?